sematic/ee/BUILD

```diff
@@ -2,8 +2,12 @@ sematic_py_lib(
     name = "ray",
     srcs = ["ray.py"],
     deps = [
+        "//sematic:init",
         "//sematic/ee/plugins/external_resource/ray:checkpoint",
         "//sematic/ee/plugins/external_resource/ray:cluster",
+        "//sematic/examples/cifar_classifier:cifar_classifier_lib",
```

Comment:
It seems strange to say that `ray` depends on these examples, rather than the other way around. If you just want to make sure these examples get included in the wheel, modify `extras.bzl` so that `EXTRA_SEMATIC_DEPS` gets these as dependencies.

Reply:
Done.